### DISCUSSION TITLE
Show UI only only on "modelISVisible"

### DISCUSSION THREAD

**Discussion by adrianposteuca** [google/model-viewer#3556] [Q&A]
Hello all,

How can an show the UI only when the model is visible?
I have this attribute on my ModelViewer 
...
loading="lazy"
reveal="interaction"
...
And I have a cycling poster for the start.
How can I find when the poster is dismissed or when the model is loaded.

I found this atributes - "loaded" and "modelIsVisible" (bool) but I can not see how they work.

Thank you

**Comment by elalish** [comment]
I think you might be looking for our related events: https://modelviewer.dev/docs/#entrydocs-loading-events-load
